test/AutoDiff/compiler_crashers/sr14223-apply-opened-opened-existential-argument.swift

```diff
@@ -1,4 +1,5 @@
-// RUN: not --crash %target-swift-frontend -disable-availability-checking -emit-sil -verify %s
+// RUN: %target-swift-frontend -disable-availability-checking -emit-sil -verify %s
+// XFAIL: *
```

Comment:
Why is this XFAIL'd? Can you just leave it as not --crash?

Reply:
Thanks. This was unintended and reverted.